src/hooks/tests/useIndexOfLastVisibleChild.test.tsx

```diff
@@ -12,7 +12,7 @@ window.ResizeObserver = window.ResizeObserver
   }));
 
 function TestComponent() {
-  const [containerElementRef, setContainerElementRef] = React.useState(null);
+  const [containerElementRef, setContainerElementRef] = React.useState<HTMLDivElement | null>(null);
   const overflowElementRef = React.useRef(null);
```

Comment:
nit/question: should both refs get the same type with `HTMLDivElement`, given both refs are applied to `divs` below?

Reply:
Done: a3fea9e291c5b3fc28ccd4be154d5d5695cd8009

Note that changing it to `useRef` instead of `useState` for consistency seems to make one of the tests fail, so I've just left one as `useState` and one as `useRef` for now, but at least they're both `HTMLDivElement` refs.